source/support.tex

```diff
@@ -542,7 +542,7 @@
 after inclusion of any member of the set of library headers
 indicated in the corresponding comment in this synopsis.
 \begin{note}
-Future revisions of \Cpp{} might replace
+Future revisions of \Cpp{} can replace
```

Comment:
I know it's necessary, but this does not seem like an improvement.

Reply:
How about rephrasing completely: "The values of these macros will increase in future revisions of \Cpp{} which modify the corresponding feature."?

Reply:
Future revisions of \Cpp{} will replace
the values of these macros with greater values
when the corresponding feature is modified.